internal/jimm/ssh/ssh.go

```diff
@@ -15,6 +19,9 @@ import (
 	"github.com/canonical/jimm/v3/internal/rpc"
 )
 
+// jujuSSHDefaultPort is the default port we expect the juju controllers to respond on.
+const jujuSSHDefaultPort = 17022
```

Comment:
Another discussion for another time, we need a "default values file". Having random scattered vars for defaults doesn't feel good.